deploy.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+STAGE=$1
+
+echo "Deploying for stage $STAGE"
```

Comment:
Noteworthy that this just uploads a fresh docker image, it doesn't really "deploy"?

Reply:
I added a task in the followup card for this. It's just meant to update the image in ECR. We could have it update the stack as well